src/get_estimation.py

```diff
@@ -695,21 +636,15 @@ def get_estimation(x, t, m, y, estimator, config):
                 calibration=True,
                 calib_method="isotonic",
             )
-        else:
-            effects = [np.nan] * 5
-    if estimator == "simulation_based":
+    elif estimator == "simulation_based":
         if config in (0, 1, 2):
             effects = r_mediate(y, t, m, x, interaction=False)
-        else:
-            effects = [np.nan] * 5
-    if estimator == "DML_huber":
+    elif estimator == "DML_huber":
         if config > 0:
             effects = medDML(y, t, m, x, trim=0.0, order=1)
-        else:
-            effects = [np.nan] * 5
-    if estimator == "G_estimator":
+    elif estimator == "G_estimator":
         if config in (0, 1, 2):
             effects = g_estimator(y, t, m, x)
-        else:
-            effects = [np.nan] * 5
+    else:
+        warnings.warn("Unrecognized estimator.")
```

Comment:
Should be an error, no ?

Reply:
It was intended to return NaN coupled with a Warning in #19, I didn't know what was the better practice between returning an error or a warning.

What kind of error should I return?

Reply:
NotImplementedError.
You need to inform the user that he's not using the code correctly.

Reply:
done, 16af6d4

Reply:
> NotImplementedError. You need to inform the user that he's not using the code correctly.

Shouldn't be a ValueError?

Reply:
yes, ValueError is fine I think.